schema.sql

```diff
@@ -53,4 +53,16 @@ CREATE TABLE specializations (
  vet_id BIGINT REFERENCES vets (id),
  animals_id BIGINT REFERENCES animals (id),
  date_of_visit  DATE
- );
\ No newline at end of file
+ );
+
+ -- PERFORMENCE TEST PART
+
+-- ADD NEW email COULUMN INSIDE TABLE owners.
+
+ ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+-- CREATE INDEXES ON COLUMNS TO SPEED UP THE PERFORMENCE.
```

Comment:
Make sure add proper database performance audit implementation for `SELECT * FROM visits where vet_id = 2;` query. I see the execution time after implementing database performance audit still greater than 1000ms.

Reply:
do you any suggetions.